vsphere_storage_plugin/cidata.py

```diff
@@ -67,7 +62,7 @@ def create(rawvolume_client, files, raw_files, datacenter_name,
     # apply raw files over files content
     if raw_files:
         for name in raw_files:
-            files[name] = ctx.get_resource(raw_files[name])
+            files[name] = get_resource(raw_files[name])
```

Comment:
Interesting to me why you decided to pass just this function from the ctx object and not the entire ctx?

Reply:
I have prepared for move code to SDK in future.